WCFDataService/Client/System/Data/Services/Client/MaterializeFromAtom.cs

```diff
@@ -103,6 +108,7 @@ internal class MaterializeAtom : IDisposable, IEnumerable, IEnumerator
             this.expectingPrimitiveValue = PrimitiveType.IsKnownNullableType(elementType);
 
             Debug.Assert(responseMessage != null, "Response message is null! Did you mean to use Materializer.ResultsWrapper/EmptyResults?");
+            this.responseMessage = responseMessage;
```

Comment:
We hold on to it so we can dispose it off when the MaterializeAtom is disposed.

Regards
Uffe

From: ZOE [mailto:notifications@github.com]
Sent: 5. september 2016 09:52
To: OData/odata.net <odata.net@noreply.github.com>
Cc: Lauesen.Uffe UFE <UFE@kmd.dk>; Mention <mention@noreply.github.com>
Subject: Re: [OData/odata.net] V3 client avoid stream copy for synchroneous requests. (#656)


In WCFDataService/Client/System/Data/Services/Client/MaterializeFromAtom.cs<https://github.com/OData/odata.net/pull/656#discussion_r77481677>:

> @@ -103,6 +108,7 @@ internal class MaterializeAtom : IDisposable, IEnumerable, IEnumerator

>              this.expectingPrimitiveValue = PrimitiveType.IsKnownNullableType(elementType);

>

>              Debug.Assert(responseMessage != null, "Response message is null! Did you mean to use Materializer.ResultsWrapper/EmptyResults?");

> +            this.responseMessage = responseMessage;

Why do we need back up responseMessage here? Seems it is not used in this file.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/OData/odata.net/pull/656/files/d4141362bcffd86ed7d5ceb80234c687501b726c#r77481677>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKq-suEofyb6YdYkl5OqAhOYi8peGOHGks5qm8ougaJpZM4JZoB4>.

Vi gør opmærksom på, at denne e-mail kan indeholde fortrolig information. Hvis du ved en fejltagelse modtager e-mailen, beder vi dig venligst informere afsender om fejlen ved at bruge svarfunktionen. Samtidig beder vi dig slette e-mailen i dit system uden at videresende eller kopiere den. Selvom e-mailen og ethvert vedhæftet bilag efter vores overbevisning er fri for virus og andre fejl, som kan påvirke computeren eller it-systemet, hvori den modtages og læses, åbnes den på modtagerens eget ansvar. Vi påtager os ikke noget ansvar for tab og skade, som er opstået i forbindelse med at modtage og bruge e-mailen.

Please note that this message may contain confidential information. If you have received this message by mistake, please inform the sender of the mistake by sending a reply, then delete the message from your system without making, distributing or retaining any copies of it. Although we believe that the message and any attachments are free from viruses and other errors that might affect the computer or it-system where it is received and read, the recipient opens the message at his or her own risk. We assume no responsibility for any loss or damage arising from the receipt or use of this message.